app/src/main/java/com/example/pokedexdesafio/domain/model/PokemonDetail.kt

```diff
@@ -0,0 +1,26 @@
+package com.example.pokedexdesafio.domain.model
```

Comment:
A business model class should not have nullable parameters, only when is required. We have here all parameters  nullable

Reply:
solved